bin/multivariate_rules.py

```diff
@@ -0,0 +1,355 @@
+        behavior_hbox.addWidget(QLabel('\tBehavior:')); behavior_hbox.addWidget( self.combobox_behavior )
+        self.layout.addLayout(behavior_hbox)
+        # Add a divider
+        divider = QFrame()
+        divider.setFrameShape(QFrame.HLine)  # Set horizontal line shape
+        divider.setFrameShadow(QFrame.Sunken)  # Set sunken shadow effect
+        self.layout.addWidget(divider)
+
+        # Behaviors sliders
+        self.layout_behavior_sliders = QVBoxLayout()
+        self.layout.addLayout(self.layout_behavior_sliders)
+        # Combox of signals to plot
+        signal_hbox_plot = QHBoxLayout()
+        signal_hbox_plot.addWidget(QLabel('Plot the signal:'))
+        self.combobox_signal_plot = QComboBox()
+        signal_hbox_plot.addWidget( self.combobox_signal_plot )
+        # Set the scroll widget as the widget inside the scroll area
+        self.scroll_area.setWidget(self.scroll_widget)
+        # Add the scroll area to the layout
+        self.layout.addWidget(self.scroll_area)
+        # Check box of axes
+        self.checkbox = QCheckBox('Fixed y-axis (min and max)')
+        self.checkbox.setCheckState(2)  # 2 corresponds to Checked state
+        self.layout.addWidget( self.checkbox )
+        # Add the plot
+        self.Figure = MainPlot(self.combobox_cell,self.combobox_behavior,self.combobox_signal_plot,self.layout_behavior_sliders, self.layout_signals, self.checkbox)
+        self.layout.addWidget( self.Figure )
+
+    def load_csv_file(self):
+        options = QFileDialog.Options()
+        file_name, _ = QFileDialog.getOpenFileName(self.CSV_loader, "Open CSV File", "", "CSV Files (*.csv)", options=options)
+        if file_name:
+            try:
+                self.CSV_loader.dataframe = pd.read_csv(file_name, header=None)
+                self.CSV_loader.file_label.setText(file_name)
+            except pd.errors.EmptyDataError:
+                self.CSV_loader.file_label.setText("CSV files is empty!")
+                return
+            self.CSV_loader.dataframe = self.CSV_loader.dataframe.rename(columns={0: "cell", 1: "signal", 2: "direction", 
+                                           3: "behavior", 4: "saturation", 5: "half_max",
+                                           6: "hill_power", 7: "dead"}) # version 2 of rules
+            # Add the base
+            self.CSV_loader.dataframe["base_behavior"] = 0
+            # print(self.CSV_loader.dataframe)
+            # Clear the combo box of cells and populate
+            self.combobox_cell.clear()
+            self.combobox_cell.addItems(self.CSV_loader.dataframe["cell"].unique().tolist())
+    def update_cells(self):
+        # Clear the combo box of behaviors and populate
+        self.combobox_behavior.clear()
+        list_behaviors = self.CSV_loader.dataframe.loc[self.CSV_loader.dataframe["cell"] == self.combobox_cell.currentText()]['behavior'].tolist()
+        self.combobox_behavior.addItems(list_behaviors)
+    def update_rules(self):
```

Comment:
<img width="318" alt="image" src="https://github.com/PhysiCell-Tools/PhysiCell-Studio/assets/22732264/1463d55e-9153-4afa-8247-3d7e744c9120">

behavior loaded in twice (possibly because two rules each targeting the same behavior?)

Reply:
Could you send me the rules that you tested?

Reply:
[cell_rules__.csv](https://github.com/PhysiCell-Tools/PhysiCell-Studio/files/15041863/cell_rules__.csv)